app/lib/nomis_client/alerts.rb

```diff
@@ -3,14 +3,21 @@
 module NomisClient
   class Alerts < NomisClient::Base
     class << self
-      def get(prison_number)
-        response(prison_number).map do |alert|
+      def get(prison_numbers)
+        get_response(nomis_offender_numbers: prison_numbers).map do |alert|
           attributes_for(alert)
         end
       end
 
-      def get_response(nomis_offender_number:)
-        NomisClient::Base.get("/bookings/offenderNo/#{nomis_offender_number}/alerts")
+      def get_response(nomis_offender_numbers:)
+        NomisClient::Base.post(
+          '/bookings/offenderNo/alerts',
+          body: nomis_offender_numbers.to_json,
+          headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+          }
```

Comment:
Hi @ngw, could the headers be set in `NomisClient::Base`? It feels like this should be consistent for all requests to NOMIS.